src/nvim/api/vim.c

```diff
@@ -133,7 +133,7 @@ String vim_command_output(String str, Error *err)
   return cstr_to_string((char *)get_vim_var_str(VV_COMMAND_OUTPUT));
 }
 
-/// Evaluates the expression str using the vim internal expression
+/// Evaluates the expression str using vim's internal expression
```

Comment:
`the vim internal expression` 

remember that neovim is vim, and for purposes of technical documentation, vim concepts are still vim concepts. We don't need to take ownership of vim concepts. nvim is vim.